### PR TITLE
Overall responsive layout 

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

make the report responsive according to Figma:

1. Add 16px margins to both side of the content when the screen is very narrow.
2. Fix the issue where white space is collapsed in the summary bar. 

image

image

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/reports/automated-checks-report.scss [hunk 1]
```diff
@@ -34,6 +34,10 @@ body {
         font-size: 17px;
         line-height: 24px;
     }
+
+    @media only screen and (max-width: 1000px) {
```
Comment (smoralesd) [inline src/DetailsView/reports/automated-checks-report.scss]: where did we get the `1000px` value? is it on figma?

Comment (haonliu) [reply src/DetailsView/reports/automated-checks-report.scss]: It is when the border of the screen meet the content plus margin.

File: src/tests/unit/tests/DetailsView/reports/components/report-sections/__snapshots__/report-body.test.tsx.snap [hunk 2]
```diff
@@ -180,6 +180,7 @@ exports[`ReportBody renders 1`] = `
         "violations": Array [],
       }
     }
+    toUtcString={[Function]}
```
Comment (peterdur) [inline src/tests/unit/tests/DetailsView/reports/components/report-sections/__snapshots__/report-body.test.tsx.snap]: Did this snapshot change leak from a different code change somewhere else?

Comment (haonliu) [reply src/tests/unit/tests/DetailsView/reports/components/report-sections/__snapshots__/report-body.test.tsx.snap]: Yes, I think it was a merge conflict.

**Pull request merged**